src/App.js

```diff
@@ -14,11 +14,68 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    good: 'Jam',
+    selected: true,
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <h1>
+          Selected good: -
+          {' '}
+          {this.state.selected
```

Comment:
you can simply check if there is selected or not, don't need to create another variable in state